### PR TITLE
EMISWEB-125 fix: error when updating event without dataValues fixed

### PR DESCRIPTION
**Pull request by cnandza**

### CODE REVIEW

File: src/hooks/tei/useGetTei.ts [hunk 1]
```diff
@@ -1,35 +1,27 @@
...
             orgUnit: orgUnit,
             trackedEntity: trackedEntity
-        })).catch((error) => {
-            show({
-                message: `${("Could not get data")}: ${error.message}`,
-                type: { critical: true }
-            });
-            setTimeout(hide, 5000);
-        }) as unknown as TeiQueryResults
+        })) as unknown as TeiQueryResults
```
Comment (EdsonNhancale) [inline src/hooks/tei/useGetTei.ts:24]: why you remove the catch function?

File: src/hooks/events/useGetEvent.ts [hunk 2]
```diff
@@ -23,7 +23,8 @@ export function useGetEvent() {
                 filter: filter,
                 orgUnit: orgUnit,
                 trackedEntity: trackedEntity
-            })) as unknown as EventQueryResults;
+            }))
+            .catch((error) => error ) as unknown as EventQueryResults;
```
Comment (EdsonNhancale) [inline src/hooks/events/useGetEvent.ts]: Why you didnt put show alert component

Comment (cnandza) [reply src/hooks/events/useGetEvent.ts]: I'm doing it on the hook where i call this function

Comment (EdsonNhancale) [reply src/hooks/events/useGetEvent.ts]: Okay, if you're doing on that way, you dont need to put catch function two times, just put one time

**Review by AlziraXerinda — APPROVED** [review]

**Review by EdsonNhancale — APPROVED** [review]

**Review by Eddy-Nhabinde — APPROVED** [review]

**Pull request merged**